language/functional_tests/tests/testsuite/generics/option.mvir

```diff
@@ -0,0 +1,31 @@
+module Option {
+    import 0x0.Vector;
```

Comment:
Yeah, this one was less interesting, but you adequately capture the issue in language/functional_tests/tests/testsuite/generics/instantiation_loops/recursive_infinite_type_terminates.mvir